{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/app.py

```diff
@@ -4,8 +4,17 @@
 import asyncio
 from aiohttp import web
 
+
 from .handlers import routes
-from . import mongo
+from .plugins import mongo, cors
+
+def setup_logging():
+    logging.basicConfig(level=logging.DEBUG)
+
+# Logger must not be initialized before it was configured
+# Or else logging handlers will not get triggered appropriately
```

Comment:
This won't work as this might be before this logger but we already imported other modules and their loggers are already created.

There is an option `disable_existing_loggers` when configuring the logging module which could be used instead. Generally I do think we should include more configurability anyway... I will make a different proposal

Reply:
See PR #2